src/test/java/org/example/PetTest.java

```diff
@@ -1,4 +1,65 @@
 package org.example;
 
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
 public class PetTest {
+
+    @Test
+    public void getNameReturnsName() {
+        Pet bello = new Pet("Bello", 20, "teckel");
+        String actual = bello.getName();
+        //assert
+        assertEquals("Bello", actual);
+    }
+
+
+    @Test
+    public void setNameOverridesName() {
+        Pet pollo = new Pet("Pollo", 20, "teckel");
+        pollo.setName("Pollo-awesome");
+        assertEquals("Pollo-awesome", pollo.getName());
+    }
+
+    @Test
+    public void getAgeReturnsAge() {
+        Pet pollo = new Pet("Pollo", 20, "teckel");
+        assertEquals(20, pollo.getAge());
+    }
+
+    @Test
+    public void setAgeOverridesAge() {
+        Pet pollo = new Pet("Pollo", 20, "teckel");
+        pollo.setAge(23);
+        assertEquals(23, pollo.getAge());
+    }
+
+    @Test
+    public void setSpeciesOverriseSpecies() {
+        Pet pollo = new Pet("Pollo", 20, "teckel");
+        pollo.setSpecies("golden retriever");
+        assertEquals("golden retriever", pollo.getSpecies());
+    }
+
+    //
+    @Test
+    public void getOwnerReturnsOwner(){
+        Person owner = new Person ("Camilo", "Muijs", "male", 34);
+        Pet abu = new Pet ("Abu", 3,"teckel");
+        abu.setOwner(owner);
+        Person testing = abu.getOwner();
+        assertEquals(owner, abu.getOwner());
```

Comment:
Hier wordt testing niet aangeroepen, Hij doet het nu doordat owner een person is maar met meerdere eigenaren doet hij het niet zoals hieronder het voorbeeld

    @Test
    public void getOwnerReturnsOwner(){
        Person owner = new Person ("Camilo", "Muijs", "male", 34);
        Person owner1 = new Person ("Susanne", "Muijs", "female", 32);
        Pet abu = new Pet ("Abu", 3,"teckel");
        abu.setOwner(owner1);
        Person testing = abu.getOwner();
        assertEquals(owner, abu.getOwner());
    }
    
        @Test
    public void getOwnerReturnsOwner(){
        Person owner = new Person ("Camilo", "Muijs", "male", 34);
        Person owner1 = new Person ("Susanne", "Muijs", "female", 32);
        Pet abu = new Pet ("Abu", 3,"teckel");
        abu.setOwner(owner1);
        Person testing = abu.getOwner();
        assertEquals(testing, abu.getOwner());
    }